### ISSUE TITLE
Conduction Plates are Stackable

### ISSUE THREAD

**Issue by Kurfursten** [d3athrow/vgstation13#34786]
Obviously this defeats the entire purpose of having more than one eel tank, so shouldn't be possible

**Comment by pedr69** [comment]
so are rad collectors, emitters, supermatter shards and basically everything else

**Comment by Kurfursten** [comment]
While I don't like those other things being stackable, those aren't really a problem. You are intended to have lots of rad collectors to one shard, for example, and stacking shards provides no benefit vs just having them nearby but separated by something to stop movement (really it just increases risk). In this case you actually are supposed to have more eels = more power. Stacking plates means you need to maintain less aquariums, exploiting to avoid more maintenance.

It's an immersion issue too. Having two shards generate more power makes sense. But conduction plates are conducting the electricity created by the eels out of the water. Adding more plates doesn't increase the amount of electricity generated by the eels. If anything, the power should be split evenly across all plates generating no additional power.

But more than that with anything, it's how pitifully easy it is. At least with the other engineering stacking you need to get a teleporter such as the hand tele to do it, so there is some investment. Conduction plates are not dense so anyone can stack them easily with no effort.

**Comment by pedr69** [comment]
>immersion issue

you're right, i just think "fixing" every instance similar to this would be very time consuming and not really worth it(apart from making the players go "_thanks c*ders, now i have to rebuild this shit 8 more times_")
>maintenance

huh? are you not using the cleaner fish? with 8 eels and 2 cleaners you can leave the tank indefinitely

**Comment by Wouju** [comment]
> > maintenance
> 
> huh? are you not using the cleaner fish? with 8 eels and 2 cleaners you can leave the tank indefinitely

maintenance could also apply to feeding the fish

**Comment by pedr69** [comment]
which you need to do for the first 13 seconds after the tank is built to get enough eggs
fish don't need food to live, only sex
